package.json

```diff
@@ -18,5 +19,19 @@
   "bugs": {
     "url": "https://github.com/NotAsBadAsItSteams/customerReviews/issues"
   },
-  "homepage": "https://github.com/NotAsBadAsItSteams/customerReviews#readme"
+  "homepage": "https://github.com/NotAsBadAsItSteams/customerReviews#readme",
+  "dependencies": {
+    "body-parser": "^1.18.2",
```

Comment:
Do you use this? If not, I suggest removing it until you do.

Reply:
Removed!